content/news/playtest-20160403.md

```diff
@@ -0,0 +1,118 @@
+We are pleased to announce the first playtest leading up to our next stable release.
+
+Our development branch currently contains more than five months worth of changes, many of them very fundamental. As such, we have finally entered the feature freeze phase of the development cycle, where we stop adding features and focus on fixing bugs found during playtesting. This is where *you* come into play! We need *your* help to give this playtest as wide a test coverage as possible to ensure another successful stable release of OpenRA! We therefore ask you to please report any and all bugs and problems you encounter on [our bug tracker](http://bugs.openra.net), in the OpenRA [IRC channel](/community/), or here in the comments. We will strive to fix as many bugs as possible before a final release. Thank you for helping to improve OpenRA!
```

Comment:
"many of them very fundamental" / "biggest changes this time around are under the hood" / "some of these changes are so fundamental" feels like you're stressing the point a bit *too* much :)